tests/e2e/network/service_annotations.go

```diff
@@ -195,6 +197,35 @@ var _ = Describe("Service with annotation", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ret).To(Equal(true), "external ip %s is not in the target subnet %s", ip, newSubnetCIDR)
 	})
+	It("should bound to the specified load balancer from azure-load-balancer-mode [MultipleAgentPools][VMSS]", func() {
+		//get nodelist and providerID specific to an agentnodes
+		nodes, err := utils.GetAgentNodes(cs)
+		Expect(err).NotTo(HaveOccurred())
+		// Get all the vmss names from all node's providerIDs
+		vmss_name := sets.NewString()
+		var resourcegrpverify string
+		for _, node := range nodes {
+			providerID := node.Spec.ProviderID
+			vmss, err := azure.ExtractScaleSetNameByProviderID(providerID)
+			Expect(err).NotTo(HaveOccurred())
+			resourceRE := regexp.MustCompile(`azure:///subscriptions/(?:.*)/resourceGroups/(.+)/providers/Microsoft.Compute/(?:.*)`)
+			resourcematch := resourceRE.FindStringSubmatch(providerID)
+			if len(resourcematch) != 2 {
+			utils.Logf("Report error")
+			return
+			}
+			resourcegrpverify = resourcematch[1]
+			vmss_name.Insert(vmss)
+		}
+		//Skip if there're less than two vmss nodes
+		if len(vmss_name) < 2 {
+			Skip("at least two vmss are required for azure-load-balancer-mode  validation")
+		}
```

Comment:
`resourcegrpverify` also needs to check here. If it is empty, then fail the test